server/value_requests.go

```diff
@@ -0,0 +1,68 @@
+package server
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strconv"
+
+	"github.com/apollo/db"
+	"github.com/labstack/echo/v4"
+)
+
+// srv.GET("/devices/:d_id/sensors/:s_id/values", getValues)
+func getValues(c echo.Context) error {
+	var start, err = strconv.Atoi(c.QueryParam("start"))
```

Comment:
It would be interesting to try the following approach

```
type (
	RangeParam struct {
		Start string `query: "start"`
		End string `query: "end"`
	}
)
...

var params RangeParam
	if err := (&echo.DefaultBinder{}).BindQueryParams(c, &params); err != nil {
		return err
	}

```